
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Documentation for the UAntwerpen-modules repository">
      
      
      
      <link rel="icon" href="../../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.2.3, mkdocs-material-8.2.5">
    
    
      
        <title>BioTools Python 2020a.00 intel 2020a Python 3.8.3 - UAntwerpen-modules documentation</title>
      
    
    
      <link rel="stylesheet" href="../../../assets/stylesheets/main.2d9f7617.min.css">
      
        
        <link rel="stylesheet" href="../../../assets/stylesheets/palette.e6a45f82.min.css">
        
      
    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../../stylesheets/extra.css">
    
    <script>__md_scope=new URL("../../..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="black" data-md-color-accent="">
  
    
    
      <script>var palette=__md_get("__palette");if(palette&&"object"==typeof palette.color)for(var key of Object.keys(palette.color))document.body.setAttribute("data-md-color-"+key,palette.color[key])</script>
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#biotools-python-2020a00-intel-2020a-python-383eb" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--lifted" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../../.." title="UAntwerpen-modules documentation" class="md-header__button md-logo" aria-label="UAntwerpen-modules documentation" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            UAntwerpen-modules documentation
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              BioTools Python 2020a.00 intel 2020a Python 3.8.3
            
          </span>
        </div>
      </div>
    </div>
    
      <form class="md-header__option" data-md-component="palette">
        
          
          
          <input class="md-option" data-md-color-media="" data-md-color-scheme="" data-md-color-primary="black" data-md-color-accent=""  aria-hidden="true"  type="radio" name="__palette" id="__palette_1">
          
        
      </form>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
  </nav>
  
    
      
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-tabs__inner md-grid">
    <ul class="md-tabs__list">
      
        
  
  


  <li class="md-tabs__item">
    <a href="../../.." class="md-tabs__link">
      Pacakge list
    </a>
  </li>

      
        
  
  


  <li class="md-tabs__item">
    <a href="../../../index.html#a" class="md-tabs__link">
      A
    </a>
  </li>

      
    </ul>
  </div>
</nav>
    
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../../.." title="UAntwerpen-modules documentation" class="md-nav__button md-logo" aria-label="UAntwerpen-modules documentation" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    UAntwerpen-modules documentation
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../.." class="md-nav__link">
        Pacakge list
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../../index.html#a" class="md-nav__link">
        A
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content" data-md-component="content">
            <article class="md-content__inner md-typeset">
              
                


<p><a href="../package/">[BioTools-Python]</a> <a href="../../..">[package list]</a></p>
<h1 id="biotools-python-2020a00-intel-2020a-python-383eb">BioTools-Python-2020a.00-intel-2020a-Python-3.8.3.eb</h1>
<div class="highlight"><pre><span></span><code><span class="n">easyblock</span> <span class="o">=</span> <span class="s1">&#39;Bundle&#39;</span>

<span class="n">local_buildset</span>         <span class="o">=</span> <span class="s1">&#39;2020a&#39;</span>
<span class="n">local_gcc_version</span>      <span class="o">=</span> <span class="s1">&#39;9.3.0&#39;</span>

<span class="n">local_Python3_version</span> <span class="o">=</span>  <span class="s1">&#39;3.8.3&#39;</span>     <span class="c1"># https://www.python.org/downloads/</span>

<span class="n">name</span> <span class="o">=</span>          <span class="s1">&#39;BioTools-Python&#39;</span>
<span class="n">version</span> <span class="o">=</span>       <span class="s1">&#39;2020a.00&#39;</span>
<span class="n">versionsuffix</span> <span class="o">=</span> <span class="s1">&#39;-Python-</span><span class="si">%(pyver)s</span><span class="s1">&#39;</span>

<span class="n">homepage</span> <span class="o">=</span> <span class="s1">&#39;(none)&#39;</span>

<span class="c1"># We list all libraries in a whatis line so that they can be found by module keyword.</span>
<span class="c1"># Contrary to what the LMOD documentation makes you believe, it seems that help is</span>
<span class="c1"># not searched when using module keyword.</span>
<span class="n">whatis</span> <span class="o">=</span> <span class="p">[</span>
    <span class="s2">&quot;Description: A number of Python bioinformatics tools, installed together as a bundle&quot;</span><span class="p">,</span>
    <span class="s2">&quot;Contains: Biopython, DLCpar, Flye, HTSeq, pysam, SICER2&quot;</span>
<span class="p">]</span>

<span class="n">local_Biopython_version</span> <span class="o">=</span> <span class="s1">&#39;1.76&#39;</span>   <span class="c1"># https://pypi.org/project/biopython/</span>
<span class="n">local_DLCpar_version</span> <span class="o">=</span>    <span class="s1">&#39;1.1&#39;</span>    <span class="c1"># https://www.cs.hmc.edu/~yjw/software/dlcpar/</span>
<span class="n">local_Flye_version</span> <span class="o">=</span>      <span class="s1">&#39;2.7.1&#39;</span>  <span class="c1"># https://github.com/fenderglass/Flye/releases</span>
<span class="n">local_HTSeq_version</span> <span class="o">=</span>     <span class="s1">&#39;0.12.4&#39;</span> <span class="c1">#https://pypi.org/project/HTSeq/</span>
<span class="n">local_Pysam_version</span> <span class="o">=</span>     <span class="s1">&#39;0.15.4&#39;</span> <span class="c1"># https://pypi.org/project/pysam/</span>
<span class="n">local_SICER2_version</span> <span class="o">=</span>    <span class="s1">&#39;1.0.2&#39;</span>  <span class="c1"># https://pypi.org/project/SICER2/</span>

<span class="n">description</span> <span class="o">=</span> <span class="s2">&quot;&quot;&quot;</span>
<span class="s2">This module provides a number of useful Python bio-informatics tools, often</span>
<span class="s2">tools that wrap around tools in the BioTools package.</span>

<span class="s2">Overview of the included tools and libraries:</span>

<span class="s2">* Biopython (version </span><span class="si">%(biopython)s</span><span class="s2">): a set of freely available tools for </span>
<span class="s2">  biological computation written in Python by an international team of </span>
<span class="s2">  developers. It is a distributed collaborative effort to develop Python </span>
<span class="s2">  libraries and applications which address the needs of current and future </span>
<span class="s2">  work in bioinformatics.</span>
<span class="s2">* DLCpar (version </span><span class="si">%(DLCpar)s</span><span class="s2">): DLCpar is a reconciliation method for inferring </span>
<span class="s2">  gene duplications, losses, and coalescence (accounting for incomplete  </span>
<span class="s2">  lineage sorting).</span>
<span class="s2">* FLye (version </span><span class="si">%(Flye)s</span><span class="s2">): a de novo assembler for single molecule sequencing reads, </span>
<span class="s2">  such as those producedby PacBio and Oxford Nanopore Technologies. It is </span>
<span class="s2">  designed for a wide range of datasets, from small bacterial projects to </span>
<span class="s2">  large mammalian-scale assemblies. The package represents a complete pipeline: </span>
<span class="s2">  it takes raw PacBio / ONT reads as input and outputs polished contigs. Flye </span>
<span class="s2">  also has a special mode for metagenome assembly.</span>
<span class="s2">* HTSeq (version </span><span class="si">%(HTSeq)s</span><span class="s2">): A framework to process and analyze data from </span>
<span class="s2">  high-throughput sequencing (HTS) arrays</span>
<span class="s2">* pysam (version </span><span class="si">%(Pysam)s</span><span class="s2">): a python module for reading and manipulating Samfiles. </span>
<span class="s2">  It&#39;s a lightweight wrapper of the samtools C-API. Pysam also includes an interface </span>
<span class="s2">  for tabix.</span>
<span class="s2">* SICER2 (version </span><span class="si">%(SICER2)s</span><span class="s2">): Redesigned and improved ChIP-seq broad peak calling tool SICER.</span>
<span class="s2">  Chromatin immunoprecipitation combined with high-throughput sequencing </span>
<span class="s2">  (ChIP-seq) can be used to map binding sites of a protein of interest in </span>
<span class="s2">  the genome. Histone modifications usually occupy broad chromatin domains </span>
<span class="s2">  and result in diffuse patterns in ChIP-seq data that make it difficult to </span>
<span class="s2">  identify signal enrichment. SICER, a spatial clustering approach for the </span>
<span class="s2">  identification of ChIP-enriched regions, was developed for calling broad </span>
<span class="s2">  peaks from ChIP-seq data.</span>
<span class="s2">  Usability of the original SICER software has been affected by increased </span>
<span class="s2">  throughputs of ChIP-seq experiments over the years. We now present SICER2 </span>
<span class="s2">  a more user-friendly version of SICER that has been redisgned and streamlined </span>
<span class="s2">  to handle large ChIP-seq data sets. This new Python package supports multiple </span>
<span class="s2">  job submissions on cluster systems and parallel processing on multicore </span>
<span class="s2">  architectures.</span>
<span class="s2">&quot;&quot;&quot;</span> <span class="o">%</span> <span class="p">{</span> 
    <span class="s1">&#39;biopython&#39;</span><span class="p">:</span> <span class="n">local_Biopython_version</span><span class="p">,</span>
    <span class="s1">&#39;DLCpar&#39;</span><span class="p">:</span>    <span class="n">local_DLCpar_version</span><span class="p">,</span>
    <span class="s1">&#39;Flye&#39;</span><span class="p">:</span>      <span class="n">local_Flye_version</span><span class="p">,</span>
    <span class="s1">&#39;HTSeq&#39;</span><span class="p">:</span>     <span class="n">local_HTSeq_version</span><span class="p">,</span>
    <span class="s1">&#39;Pysam&#39;</span><span class="p">:</span>     <span class="n">local_Pysam_version</span><span class="p">,</span>
    <span class="s1">&#39;SICER2&#39;</span><span class="p">:</span>    <span class="n">local_SICER2_version</span><span class="p">,</span>
<span class="p">}</span>

<span class="n">docurls</span> <span class="o">=</span> <span class="p">[</span>
    <span class="s1">&#39;biopython: Web-based information on https://biopython.org/&#39;</span><span class="p">,</span>
    <span class="s1">&#39;DLCpar:    See https://www.cs.hmc.edu/~yjw/software/dlcpar/&#39;</span><span class="p">,</span>
    <span class="s1">&#39;Flye:      Web-based manual on https://github.com/fenderglass/Flye/blob/flye/docs/USAGE.md&#39;</span><span class="p">,</span>
    <span class="s1">&#39;HTSeq:     Web-based documentation on https://htseq.readthedocs.io/&#39;</span><span class="p">,</span>
    <span class="s1">&#39;Pysam:     Web-based documentation on https://pysam.readthedocs.io/en/latest/&#39;</span><span class="p">,</span>
    <span class="s1">&#39;SICER2:    Web-based documentation on https://zanglab.github.io/SICER2/&#39;</span><span class="p">,</span>
<span class="p">]</span>

<span class="n">toolchain</span> <span class="o">=</span> <span class="p">{</span><span class="s1">&#39;name&#39;</span><span class="p">:</span> <span class="s1">&#39;intel&#39;</span><span class="p">,</span> <span class="s1">&#39;version&#39;</span><span class="p">:</span> <span class="n">local_buildset</span><span class="p">}</span>

<span class="n">dependencies</span> <span class="o">=</span> <span class="p">[</span>
    <span class="p">(</span><span class="s1">&#39;Python&#39;</span><span class="p">,</span>   <span class="n">local_Python3_version</span><span class="p">),</span>
    <span class="p">(</span><span class="s1">&#39;BioTools&#39;</span><span class="p">,</span> <span class="n">version</span><span class="p">),</span>  
<span class="p">]</span>

<span class="n">default_easyblock</span> <span class="o">=</span> <span class="s2">&quot;Tarball&quot;</span>
<span class="n">default_component_specs</span> <span class="o">=</span> <span class="p">{</span>
    <span class="s1">&#39;skipsteps&#39;</span><span class="p">:</span> <span class="p">[</span><span class="s1">&#39;install&#39;</span><span class="p">],</span>
<span class="p">}</span>

<span class="n">exts_defaultclass</span> <span class="o">=</span> <span class="s1">&#39;PythonPackage&#39;</span>
<span class="n">exts_filter</span> <span class="o">=</span> <span class="p">(</span><span class="s2">&quot;python -c &#39;import </span><span class="si">%(ext_name)s</span><span class="s2">&#39;&quot;</span><span class="p">,</span> <span class="s1">&#39;&#39;</span><span class="p">)</span>
<span class="n">exts_default_options</span> <span class="o">=</span> <span class="p">{</span>
    <span class="s1">&#39;download_dep_fail&#39;</span><span class="p">:</span> <span class="kc">True</span><span class="p">,</span>
    <span class="s1">&#39;sanity_pip_check&#39;</span><span class="p">:</span>  <span class="kc">False</span><span class="p">,</span>
    <span class="s1">&#39;source_urls&#39;</span><span class="p">:</span>       <span class="p">[</span><span class="n">PYPI_SOURCE</span><span class="p">],</span>
    <span class="s1">&#39;use_pip&#39;</span><span class="p">:</span>           <span class="kc">True</span><span class="p">,</span>
<span class="p">}</span>

<span class="n">components</span> <span class="o">=</span> <span class="p">[</span>    
    <span class="p">(</span><span class="s1">&#39;Flye&#39;</span><span class="p">,</span> <span class="n">local_Flye_version</span><span class="p">,</span> <span class="p">{</span>
        <span class="s1">&#39;sources&#39;</span> <span class="p">:</span>  <span class="p">[{</span>
                       <span class="s1">&#39;download_filename&#39;</span><span class="p">:</span> <span class="s1">&#39;</span><span class="si">%(version)s</span><span class="s1">.zip&#39;</span><span class="p">,</span> 
                       <span class="s1">&#39;filename&#39;</span><span class="p">:</span> <span class="s1">&#39;</span><span class="si">%(namelower)s</span><span class="s1">-</span><span class="si">%(version)s</span><span class="s1">.zip&#39;</span><span class="p">,</span> 
                       <span class="s1">&#39;source_urls&#39;</span><span class="p">:</span> <span class="p">[</span><span class="s1">&#39;https://github.com/fenderglass/Flye/archive/&#39;</span><span class="p">]</span> 
                     <span class="p">}],</span>
        <span class="s1">&#39;start_dir&#39;</span><span class="p">:</span> <span class="s1">&#39;</span><span class="si">%(name)s</span><span class="s1">-</span><span class="si">%(version)s</span><span class="s1">&#39;</span><span class="p">,</span>
    <span class="p">}),</span>   
<span class="p">]</span>

<span class="n">exts_list</span> <span class="o">=</span> <span class="p">[</span>
    <span class="p">(</span><span class="s1">&#39;biopython&#39;</span><span class="p">,</span> <span class="n">local_Biopython_version</span><span class="p">,</span> <span class="p">{</span>
        <span class="s1">&#39;modulename&#39;</span><span class="p">:</span> <span class="s1">&#39;Bio&#39;</span><span class="p">,</span>
    <span class="p">}),</span>   
    <span class="p">(</span><span class="s1">&#39;flye&#39;</span><span class="p">,</span> <span class="n">local_Flye_version</span><span class="p">,</span> <span class="p">{</span>
        <span class="s1">&#39;source_tmpl&#39;</span><span class="p">:</span> <span class="s1">&#39;flye-</span><span class="si">%(version)s</span><span class="s1">.zip&#39;</span><span class="p">,</span>
    <span class="p">}),</span>   
    <span class="p">(</span><span class="s1">&#39;pysam&#39;</span><span class="p">,</span> <span class="n">local_Pysam_version</span><span class="p">,</span> <span class="p">{</span>
        <span class="c1"># &#39;preinstallopts&#39;: &#39;HTSLIB_LIBRARY_DIR=$EBROOTBIOTOOLS/lib HTSLIB_INCLUDE_DIR=$EBROOTBIOTOOLS/include &#39;,</span>
    <span class="p">}),</span>   
    <span class="p">(</span><span class="s1">&#39;SICER2&#39;</span><span class="p">,</span> <span class="n">local_SICER2_version</span><span class="p">,</span> <span class="p">{</span>
        <span class="s1">&#39;modulename&#39;</span><span class="p">:</span> <span class="s1">&#39;sicer&#39;</span><span class="p">,</span>
    <span class="p">}),</span>
    <span class="p">(</span><span class="s1">&#39;HTSeq&#39;</span><span class="p">,</span> <span class="n">local_HTSeq_version</span><span class="p">,</span> <span class="p">{</span>
        <span class="s1">&#39;modulename&#39;</span><span class="p">:</span> <span class="s1">&#39;HTSeq&#39;</span><span class="p">,</span>
    <span class="p">}),</span>
<span class="c1">#</span>
<span class="c1"># Considered for removal in future versions</span>
<span class="c1">#</span>
    <span class="p">(</span><span class="s1">&#39;DLCpar&#39;</span><span class="p">,</span> <span class="n">local_DLCpar_version</span><span class="p">,</span> <span class="p">{</span>
        <span class="s1">&#39;source_tmpl&#39;</span><span class="p">:</span> <span class="s1">&#39;</span><span class="si">%(namelower)s</span><span class="s1">-</span><span class="si">%(version)s</span><span class="s1">.tar.gz&#39;</span><span class="p">,</span>
        <span class="s1">&#39;source_urls&#39;</span><span class="p">:</span> <span class="p">[</span><span class="s1">&#39;https://www.cs.hmc.edu/~yjw/software/dlcpar/pub/sw&#39;</span><span class="p">]</span>
    <span class="p">})</span>
<span class="p">]</span>

<span class="n">sanity_check_paths</span> <span class="o">=</span> <span class="p">{</span>
    <span class="s1">&#39;files&#39;</span><span class="p">:</span> <span class="p">[],</span>
    <span class="s1">&#39;dirs&#39;</span><span class="p">:</span>  <span class="p">[</span><span class="s1">&#39;lib/python</span><span class="si">%(pyshortver)s</span><span class="s1">/site-packages&#39;</span><span class="p">],</span>
<span class="p">}</span>

<span class="n">modextrapaths</span> <span class="o">=</span> <span class="p">{</span>
    <span class="s1">&#39;PYTHONPATH&#39;</span><span class="p">:</span> <span class="s1">&#39;lib/python</span><span class="si">%(pyshortver)s</span><span class="s1">/site-packages&#39;</span><span class="p">,</span>   
<span class="p">}</span>

<span class="n">moduleclass</span> <span class="o">=</span> <span class="s1">&#39;bio&#39;</span>
</code></pre></div>
<p><a href="../package/">[BioTools-Python]</a> <a href="../../..">[package list]</a></p>

              
            </article>
          </div>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        <div class="md-social">
  
    
    
    <a href="https://github.com/hpcuantwerpen" target="_blank" rel="noopener" title="HPC-UAntwerpen on GitHub" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    <script id="__config" type="application/json">{"base": "../../..", "features": ["navigation.instant", "navigation.tracking", "navigation.tabs", "navigation.tabs.sticky", "navigation.indexes", "header.autohide", "toc.follow", "content.code.annotate", "search.suggest"], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../../../assets/javascripts/workers/search.bd0b6b67.min.js"}</script>
    
    
      <script src="../../../assets/javascripts/bundle.467223ff.min.js"></script>
      
    
  </body>
</html>